docs/config.md

```diff
@@ -0,0 +1,59 @@
+## Configuring Google Spreadsheet Operations
+
+To use the Google Spreadsheet connector, add the `<googlespreadsheet.init>` element in your proxy configuration before use any other
+Google Spreadsheet connector operations. This `<googlespreadsheet.init>` element used to authenticates  the user using OAuth2 authentication
+to access the Google account which contains the spreadsheets. For more information on authorizing requests in Google Spreadsheets, see [API Doc](https://developers.google.com/sheets/api/guides/authorizing).
+
+<br/>
+
+> Follow this [guide](https://docs.wso2.com/display/IntegrationCloud/Get+Credentials+for+Google+Spreadsheet) to set up the Google Spreadsheets and get credentials such as clientId, clientSecret, accessToken, refreshToken.
+
+<br/>
+
+**init**
+```xml
+<googlespreadsheet.init>
+    <accessToken>{$ctx:accessToken}</accessToken>
+    <clientId>{$ctx:clientId}</clientId>
+    <clientSecret>{$ctx:clientSecret}</clientSecret>
+    <refreshToken>{$ctx:refreshToken}</refreshToken>
+    <apiUrl>{$ctx:apiUrl}</apiUrl>
+</googlespreadsheet.init>
+```
+
+<br/>
+
+To get the OAuth access token directly call **init** (this method call getAccessTokenFromRefreshToken method itself )  method or add  `<googlespreadsheet.getAccessTokenFromRefreshToken>` element before `<googlespreadsheet.init>` element in your configuration.
+
+<br/>
+
+**getAccessTokenFromRefreshToken**
+```xml
+<googlespreadsheet.getAccessTokenFromRefreshToken>
+    <clientId>{$ctx:clientId}</clientId>
+    <clientSecret>{$ctx:clientSecret}</clientSecret>
+    <refreshToken>{$ctx:refreshToken}</refreshToken>
+</googlespreadsheet.getAccessTokenFromRefreshToken>
+```
+
+<br/>
+
+>> **Note:** For the very first time we need to use valid accessToken to use the connector operations. If the provided accessToken is expired then the token refreshing flow will be handled inside the connector.
```

Comment:
**Note:** When trying it out for the first time, you need to use a valid accessToken to use the connector operations. If the accessToken is expired then the token refreshing flow will be handled inside the connector.